src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person as PersonType } from '../../types';
+
+type Props = {
+  person: PersonType,
+  findSlug: (name:string) => string | undefined,
+};
+
+export const PersonLink: FC<Props> = ({
```

Comment:
It's not a link, as it contains all user info and renders a table row instead of a link. You can name just as person or person row